docs/src/snap/howto/security/index.md

```diff
@@ -12,7 +12,7 @@ harden their clusters in accordance with DISA STIG and CIS recommendations.
 :glob:
 :titlesonly:
 
-CIS Hardening <cis-hardening.md>
-DISA STIG Hardening <disa-stig-hardening.md>
-Hardening guide <auditing-steps.md>
+Post deployment hardening guide <hardening-post.md>
```

Comment:
I'm not sure where this is supposed to point?

There's no `hardening-post.md` anywhere (yet...)

Reply:
It's for hardening.md